flake.nix

```diff
@@ -27,6 +27,9 @@
               extensions = [ "rust-src" ];
               targets = [ "x86_64-unknown-linux-gnu" ];
             })
+            # (rust-bin.beta.latest.default.override {
+            #   targets = [ "x86_64-unknown-linux-gnu" ];
+            # })
```

Comment:
Was this intentional?

Reply:
Yes, unless you have a better way to test clippy with beta?

Reply:
No that sounds good